Compiler/FrontEnd/Expression.mo

```diff
@@ -7966,6 +7967,10 @@ algorithm
         if res then isConstWork(e1) else false;
 
     case (DAE.CALL(expLst=ae, attr=DAE.CALL_ATTR(builtin=false, isImpure=false))) then isConstWorkList(ae);
+    case (DAE.CALL(path=path, expLst=ae, attr=DAE.CALL_ATTR(builtin=true))) then
+      if true or listMember(Absyn.pathFirstIdent(path),
```

Comment:
why `if true` ???

Reply:
That's my old debugging code to see if it triggered :D